MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1604,11 +1594,30 @@ public string GetGraphicalSubtitleParam(EncodingJobInfo state, EncodingOptions o
                     state.VideoStream.Width.Value,
                     state.VideoStream.Height.Value);
 
-                //For QSV, feed it into hardware encoder now
+                // For QSV, feed it into hardware encoder now
                 if (string.Equals(outputVideoCodec, "h264_qsv", StringComparison.OrdinalIgnoreCase))
                 {
                     videoSizeParam += ",hwupload=extra_hw_frames=64";
                 }
+
+                // For VAAPI and CUVID decoder
```

Comment:
The `cuvid` decoder uses `-resize` to change the video input resolution during input, but unlike the `libx264` software encoder, which can automatically adjust the size of graphic subtitles to fit the output video, the `nvenc` encoder needs to be manually adjusted. Otherwise, the graphic subtitles will be out of view.

Reply:
would be nice to have this explanation as a comment in code for future us